packages/DateRangePicker/stories/DateRangePicker.stories.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { boolean, select, text, withKnobs } from "@storybook/addon-knobs";
+import { storiesOf } from "@storybook/react";
+import { getStoryName } from "storybook/storyTree";
+import moment from "moment";
+import L10n from "@paprika/l10n";
+import { DateRangePickerStory } from "./DateRangePicker.stories.styles";
+import DateRangePicker from "../src";
+
+const storyName = getStoryName("DateRangePicker");
+
+storiesOf(storyName, module)
+  .addDecorator(withKnobs)
+  .add("Showcase", () => {
+    const dateRangePickerProps = {
+      dateFormat: select("dateFormat", ["MM/DD/YYYY", "DD/MM/YYYY", "YYYY-MM-DD"], "MM/DD/YYYY"),
+      humanFormat: select("humanFormat", ["MMMM DD, YYYY", "YYYY-MM-DD"], "MMMM DD, YYYY"),
+      isDisabled: boolean("isDisabled", false),
+      isReadOnly: boolean("isReadOnly", false),
+      hasError: boolean("hasError", false),
+    };
+
+    const inputProps = {
+      size: select("DateRangePicker.Input - size", [undefined, "small", "medium", "large"], undefined),
+    };
+    const startInputProps = {
+      placeholder: text("DateRangePicker.StartInput - placeholder", "") || undefined,
+    };
+    const endInputProps = {
+      placeholder: text("DateRangePicker.EndInput - placeholder", "") || undefined,
+    };
+
+    const [{ startDate, endDate }, setDates] = React.useState({ startDate: null, endDate: null });
+
+    moment.locale("en");
+
+    return (
+      <L10n locale="en">
+        <DateRangePickerStory>
+          <DateRangePicker startDate={startDate} endDate={endDate} onDatesChange={setDates} {...dateRangePickerProps}>
```

Comment:
I know this allowed the author to write more stories faster and with less duplication, but the downside is that if I'm a consumer I can't understand without digging into the code how to use this component, I would encourage maybe to add a simple story without spreading showcasing how to use it. What do you think? 

Also you can create and github issue asking someone else to write the stories we can asked in a starling day to ux-dev to help us with this. I think its time to merge and keep moving into new features :)